roles/cifmw_cephadm/tasks/configure_object.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Check for exisiting object store config
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+  delegate_to: localhost
+  when: cifmw_openshift_kubeconfig is defined
+  block:
+    - name: Check if swift is enabled in deployed controlplane
+      ansible.builtin.shell: "set -o pipefail && oc get $(oc get oscp -n openstack -o name) -o json| jq .spec.swift.enabled"
+      register: swift_in_ctlplane
+    # checking swift_endpoints_count will avoid unnecessary errors during ceph deployment re-run
+    - name: Check if swift endpoint is already created
+      ansible.builtin.shell: "set -o pipefail && oc rsh openstackclient openstack endpoint list | grep 'swift.*object-store' | wc -l"
+      register: swift_endpoints_count
+      ignore_errors: true
```

Comment:
This line was my only concern, but I can't think about a wrong case because I think, worst case scenario, the `Configure object store to use rgw` is skipped

Reply:
if the swift endpoints are not configured, exit status of the command is non-zero so it may be considered as a failure.